src/server.ts

```diff
@@ -1,27 +1,42 @@
-import express, { Request, Response, ErrorRequestHandler } from 'express'
-import { Person } from './types'
+import express, { Request, Response, ErrorRequestHandler } from 'express';
+import { Person } from './types';
+import fetch from 'node-fetch';
+import _, { omitBy, isEmpty } from 'lodash';
```

Comment:
```suggestion
import _ from 'lodash';
```
then use `_.` everywhere in the code